Firestore/Source/API/FIRBsonBinaryData.mm

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Firestore/Source/Public/FirebaseFirestore/FIRBsonBinaryData.h"
+
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
nit: remove `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END` from .m or .mm files. These macros only have an effect when used in header files.

Reply:
Done.